Algorithm_No3.php

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<body>
+  <?php
+
+  function Pair($arr,$num){
+
+    $bool = FALSE;
+
+    for ($i = 0;$i < count($arr);$i++){
```

Comment:
Big-O is n^2. It is not good.
Please review my solution with Big-O is n.
Input: {11, 15, 6, 8, 9, 10} - x = 19
Step 1: Detecting routing position. (index = 2)
Step 2: Find the match sum number by logic loop : max + min
2.1 - Max + Min = arr[1] + arr[2] = 15 + 6 =21 > 19 -> down max
2.2 - Max + Min = arr[0] + arr[2] =11 + 6 = 17 < 19 -> up min
2.3 - Max + Min = arr[0] + arr[3] = 11 + 8 = 19 -> match

Reply:
it changed. Thank you for your confirmation.